packages/core/src/components/upload/upload.tsx

```diff
@@ -58,6 +58,11 @@ export class Upload {
    */
   @Prop() selectFileText = '+ Drag files here or…';
 
+  /**
+   * Label for Select file text
+   */
+  @Prop() i18nSelectFileText = '+ Drag files here or…';
```

Comment:
What is your attention behind this change?

Reply:
I want to pass this string as per selected language, similar to i18nUploadFile button internationalization.
Kindly let me know if any other details are needed.

Reply:
Can I rename `selectFileText` to `i18nSelectFileText` instead of adding new property?

Reply:
At least not in a minor version this have to be done on a major version e.g `release-3.0.0` branch.

Reply:
Ok then can you please suggest how should we proceed with this PR?
Should I close this PR and raise a defect to address this or should I follow any other steps?

Reply:
I see two options here:

Either close the PR and add the `i18n` prefix at a later point in time, or
deprecate the `selectFileText` property (and add a changeset to this PR).